config/initializers/zoo_stream.rb

```diff
@@ -2,9 +2,9 @@
   require 'aws-sdk'
   require 'zoo_stream'
   Aws.config.update region: 'us-east-1'
-  ZooStream.source = "talk-#{ Rails.env }"
+  ZooStream.source = ENV.fetch("ZOO_STREAM_SOURCE") || "talk"
```

Comment:
The syntax you were probably thinking of is the two argument method: `ENV.fetch('ZOO_STREAM_SOURCE', 'talk')` to provide a default value.